base/show.jl

```diff
@@ -1114,7 +1114,7 @@ function print_fullname(io::IO, m::Module)
     else
         print_fullname(io, mp)
         print(io, '.')
-        print(io, nameof(m))
+        show_sym(io, nameof(m))
```

Comment:
Should we apply this on line 1113 as well?

Reply:
I added it, but there seem no way to create a module with itself as a parent from julia to test it.